src/components/Clients/DeleteClientModal.jsx

```diff
@@ -0,0 +1,109 @@
+// React Imports
+import React, { useContext } from 'react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+// Utility Imports
+import { runNotification } from '../../utils';
+// Custom Hook Imports
+import { useStatusNotification } from '../../hooks';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import FormSection from '../Form/FormSection';
+
+/**
+ * DeleteClientModal Component - Component that allows users to delete other user's
+ * Pod URLs from a user's list stored on their own Pod
+ *
+ * @memberof Clients
+ * @name DeleteClientModal
+ */
+
+const DeleteClientModal = ({
+  showDeleteClientModal,
+  setShowDeleteClientModal,
+  selectedClientToDelete
+}) => {
+  const { state, dispatch } = useStatusNotification();
+  const { removeUser } = useContext(UserListContext);
+
+  // Event handler for deleting client from client list
+  const handleDeleteClient = async (event) => {
+    event.preventDefault();
+    runNotification(
+      `Deleting "${selectedClientToDelete?.person}" from client list...`,
+      5,
+      state,
+      dispatch
+    );
+    try {
```

Comment:
This block seems odd to me. It says "attempt to delete the client, then, regardless of if the attempt succeeds or fails, tell the client that it succeeded." I think what you mean is something more like this:

```
try {
  await removeUser();
  runNotification("User deleted!", 5, state, dispatch)
} catch (e) {
  runNotification(`User Deletion failed. Reason: ${e.message}`)
} finally {
  setTimeout(() => {setShowDeleteClientModal(false)}, 2000)
}
```

That says "try to delete the user. If you succeed, display a success message. If you fail, display an error message. No matter what, drop the modal after 2 seconds"

Reply:
oh yes, agree it needs a `catch` block.

But leaving this before the `try` block:
`runNotification(
      'Deleting "${selectedClientToDelete?.person}" from client list...',
      5,
      state,
      dispatch
    );`
    so the user has some sort of indication that something is going on.

Reply:
Works for me!